Games/types/Mafia/effects/Love.js

```diff
@@ -0,0 +1,10 @@
+const Effect = require("../Effect");
+
+module.exports = class Love extends Effect {
+
+    constructor(matchmaker) {
+        super("Love");
+        this.matchmaker = matchmaker;
```

Comment:
can i know what this.matchmaker does? o:

is the intention to make multiple matchmakers win separately e.g. i can only win if i personally matchmade them

Reply:
Book keeping incase theirs another role that cares about who applied an effect